src/server/mock-data/index.ts

```diff
@@ -5,7 +5,7 @@ import AKTES from './json/aktes.json';
 import BAG from './json/bag.json';
 import BAG2 from './json/bag2.json';
 import BELASTINGEN from './json/belasting.json';
-import BRP from './json/brp.json';
+import BRP from './json/brp';
```

Comment:
Waarom heb je hier naar .ts veranderd? JSON en TS hebben heel verschillende doelen. JSON is hier bedoeld als data simulatie uit de api's. Indien je ts gebruikt spoort dat het gebruik van coding logica aan. Dat wijkt af van de algemene implementatie van api simulatie.

Reply:
Ik heb deze hersteld.